device/iot-device-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/device/DeviceTwin/PropertyTest.java

```diff
@@ -68,33 +68,6 @@ public void constructorWithMetadataThrowsOnNullKey()
 
     }
 
-    /*
-    **Tests_SRS_Property_25_006: [**If the key contains illegal unicode control characters i.e ' ', '.', '$', the constructor shall throw an IllegalArgumentException.**]**
-     */
-    @Test (expected = IllegalArgumentException.class)
-    public void constructorThrowsOnInvalidKey()
-    {
-        //act
-        Property testProp = new Property("Key with space", 1);
-
-    }
-
-    @Test (expected = IllegalArgumentException.class)
-    public void constructorThrowsOnInvalidKey_2()
-    {
-        //act
-        Property testProp = new Property("KeyWith$", 1);
-
```

Comment:
Also shoudn't this commit go on master? Why is it deviating from master ?

Reply:
It did go into master already.